app/assets/javascripts/components/dashboard/navigation/navigation.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from 'react'
+import ReactDOM from 'react-dom';
+import Panes from './panes.jsx';
+import Tabs from './tabs.jsx';
+
+export default class Navigation extends Component {
+  constructor(props) {
+    super(props);
+    this.handleTabs = this.handleTabs.bind(this);
```

Comment:
Binding to `this` within the constructor is the recommended way when passing down the function to its children.